packages/core/config.d.ts

```diff
@@ -30,6 +30,15 @@ export interface Config {
      * @visibility frontend
      */
     title?: string;
+
+    /**
+     * The support url for the app.
+     * @visibility frontend
+     */
+    support?: {
```

Comment:
Should this be a list just like links? Feels likely to be useful. Could even support both if we wanted, even though the code to read it becomes slightly more clumsy.

Reply:
Oh and should it have one more level of nesting for future extension?

```yaml
support:
  description: If you need support, reach out to...
  links:
    - url: ...
```

Or should we just continue putting `children` to use for everything besides links?

Reply:
I've updated the config structure to support a grouping of `items` and within this contains a list of `links`.